.rubocop.yml

```diff
@@ -6,4 +6,12 @@ LineLength:
 
 Style/FrozenStringLiteralComment:
   Enabled: false
-  
\ No newline at end of file
+  
+Metrics/ModuleLength:
+  Exclude:
+    - "**/*_spec.rb"
```

Comment:
exclude from all cops the rails_helper and spec_helper, but we want to run rubocop in the specs